apps/allocations/app/components/App/AllocationsHistory.js

```diff
@@ -117,21 +120,31 @@ const Amount = styled.div`
   font-weight: 600;
 `
 
-// const Recipient = styled.div`
-//   display: flex;
-//   flex-direction: column;
-// `
-//
-// const RecipientProgress = styled.div`
-//   margin-top: 7px;
-//   margin-bottom: 4px;
-// `
-//
-// const RecipientAmount = styled.div`
-//   color: ${({ theme }) => theme.contentSecondary};
-//   align-self: flex-end;
-//   font-size: 12px;
-// `
+const Recipient = styled.div`
+   display: flex;
+   flex-direction: row;
+   width: 100%;
+   padding: 10px 10px 4px;
+   border-radius: 4px;
+   background-color: white;
```

Comment:
We prefer to use `theme.colorDefinition` from `aragon-ui` theme palette, instead of hardcoding the color.
The main reason is to allow, for example, switching to dark or high contrast mode, also better updates on themes.

Reply:
Understood. As I said above, I'll be removing this css to fit with the original design